tests/test-200.txt

```diff
@@ -0,0 +1,13 @@
+#!/bin/ksh

```

Comment:
It's somewhat strange for shell scripts to have `.txt` file name suffix. The `.txt` must be either changed or dropped.